schemas/context/emailaddress.schema.json

```diff
@@ -75,5 +75,5 @@
       "$ref": "https://ns.adobe.com/xdm/common/auditable#/definitions/auditlog"
     }
   ],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
👍

Reply:
👎

Remove reference to auditable, since that isn't stable.